tests/java/src/Main.java

```diff
@@ -0,0 +1,5 @@
+public class Main {
```

Comment:
Codacy found an issue: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/gh/cnescatlab/docker-cat/pullRequest?prid=6196008)